gtk4/Gir.toml

```diff
@@ -1513,6 +1512,13 @@ generate_builder = true
     name = "set_expression"
     manual = true # use AsRef<Expression>
 
+[[object]]
+name = "Gtk.Ordering"
+status = "generate"
+    [[object.function]]
+    name = "from_cmpfunc"
+    ignore = true # not useful for Rust, conversion From/Into cmp::Ordering is possible
```

Comment:
Also ignore it in the `sys` crate.

The problem is that otherwise things won't like with MSVC for people using GTK 4.0, even if the functions in question are not actually used!